# Bump Sensor information message from DrRobot Robot.

Header header    	# timestamp in the header is the time
uint8 bump_sw1		# middle_right
uint8 bump_sw2		# right
uint8 bump_sw3		# middle_left
uint8 bump_sw4		# left
